client/src/app/pages/migration-waves/migration-waves.tsx

```diff
@@ -137,14 +139,37 @@ export const MigrationWaves: React.FC = () => {
       title: getAxiosErrorMessage(error),
       variant: "danger",
     });
-    refetch();
   };
 
   const { mutate: deleteWave } = useDeleteMigrationWaveMutation(
     onDeleteWaveSuccess,
     onDeleteWaveError
   );
 
+  const onDeleteAllMigrationWavesSuccess = (response: any) => {
+    pushNotification({
+      title: t("toastr.success.deleted", {
```

Comment:
Success message currently should read 

Migration wave(s) was successfully deleted.

or if multiple: 

Migration waves were successfully deleted.

Reply:
What do we do in the case of only 1 wave slated for delete? The current form would lead to the wrong message displayed. 😅